churchslavonic.tex

```diff
@@ -2,9 +2,10 @@
 
 \newfontfamily\englishfont[Script=Latin,Ligatures=TeX,HyphenChar=-]{Linux Libertine O}
 \newfontfamily\russianfont[Script=Cyrillic,Ligatures=TeX]{Linux Libertine O}
-\newfontfamily\russianfonttt[Ligatures=TeX]{lmmono10-regular.otf}
+\setmonofont[Scale=MatchLowercase]{FiraSlav}
+\newfontfamily\russianfonttt[Scale=MatchLowercase]{FiraSlav}
 \newfontfamily\russianfontsf[Ligatures=TeX]{lmsans10-regular.otf}
-\newfontfamily\churchslavonicfont[Script=Cyrillic,Ligatures=TeX,HyphenChar=_]{PonomarUnicode.otf}
+\newfontfamily\churchslavonicfont[Script=Cyrillic,Ligatures=TeX,Scale=MatchLowercase,HyphenChar=_]{PonomarUnicode.otf}
```

Comment:
what MatchLowercase is doing? Just curious...

Reply:
It scales the Church Slavonic font so that a lowercase Church Slavonic letter is the same height as a lowercase Latin character.